opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -70,12 +70,18 @@ option go_package = "go.opentelemetry.io/proto/otlp/profiles/v1development";
 // ┌──────────────────┐   1-n   ┌─────────────────┐   ┌──────────┐
 // │      Sample      │ ──────▷ │ KeyValueAndUnit │   │   Link   │
 // └──────────────────┘         └─────────────────┘   └──────────┘
-//   │                    1-n       △      △
-//   │ 1-n        ┌─────────────────┘      │ 1-n
-//   ▽            │                        │
-// ┌──────────────────┐   n-1   ┌──────────────┐
-// │     Location     │ ──────▷ │   Mapping    │
-// └──────────────────┘         └──────────────┘
+//   │                              △      △
+//   │ 1-1                          │      │ 1-n
+//   ▽                              │      │
+// ┌──────────────────┐             │      │
+// │      Stack       │             │      │
+// └──────────────────┘             │      │
+//   │                     1-n      │      │
+//   │ 1-n         ┌────────────────┘      │
+//   ▽             │                       │
+// ┌──────────────────┐   n-1   ┌─────────────┐
```

Comment:
Saying `n-1` for Location -> Mapping and `1-1` for Sample -> Stack seems inconsistent?

It's true that multiple locations can point to the same mapping, but it's also true that multiple samples can point to the same stack?

Reply:
I switched it to `n-1` but there might be other inconsistencies (e.g. Line -> Function) that I didn't touch and that we should open a separate PR for.